package.nls.json

```
@@ -437,11 +437,16 @@
     "AWS.s3.downloadFile.error.general": "Failed to download file {0}",
     "AWS.s3.promptTime.placeHolder": "Defaults to 15 minutes",
     "AWS.s3.promptTime.prompt": "Enter a valid time (in minutes) for URL: ${0}",
+    "AWS.s3.uploadFile.progressTitle.batch": "Uploading {0} files to {1}",
     "AWS.s3.uploadFile.progressTitle": "Uploading {0}...",
+    "AWS.s3.uploadFile.progressReport": "{0}/{1} file(s) uploaded to {3}",
     "AWS.s3.uploadFile.openButton": "Upload",
     "AWS.s3.uploadFile.startUpload": "Uploading file from {0} to {1}",
-    "AWS.s3.uploadFile.success": "Successfully uploaded file {0} to {1}",
-    "AWS.s3.uploadFile.error.general": "Failed to upload file {0}",
+    "AWS.s3.uploadFile.success": "Successfully uploaded {0}/{1} file(s)",
```

"Successfully" is noise here.
```suggestion
    "AWS.s3.uploadFile.success": "Uploaded {0}/{1} file(s)",
```